src/main/java/org/jenkinsci/plugins/electricflow/Configuration.java

```diff
@@ -38,39 +39,38 @@
 
     private final String configurationName;
     private final String electricFlowUser;
-    private final String electricFlowPassword;
+    private final Secret electricFlowPassword;
     private final String electricFlowUrl;
     private final String electricFlowApiVersion;
     private final boolean ignoreSslConnectionErrors;
 
     //~ Constructors -----------------------------------------------------------
 
+    @Deprecated
+    public Configuration(
+          String configurationName,
+          String electricFlowUrl,
+          String electricFlowUser,
+          String electricFlowPassword,
+          String electricFlowApiVersion,
+          boolean ignoreSslConnectionErrors)
+    {
+        this(configurationName, electricFlowUrl, electricFlowUser, Secret.fromString(electricFlowPassword), electricFlowApiVersion, ignoreSslConnectionErrors);
+    }
+
     @DataBoundConstructor public Configuration(
             String configurationName,
             String electricFlowUrl,
             String electricFlowUser,
-            String electricFlowPassword,
+            Secret electricFlowPassword,
```

Comment:
You didn't have to change any call to this constructor? Is it because it's done through the @DataBound annotation?

Reply:
no there is no caller, as it's only coming from configurations. So done with the `@DataBoundConstructor`.